VaticanBAV/borglat/BAVborglat285.xml

```diff
@@ -25,8 +25,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msIdentifier>
                         <repository ref="INS0003BAV"/>
                         <collection>Borgiani latini</collection>
-                        <idno>Vat. Borg. lat. 285</idno>
+                        <idno facs="https://digi.vatlib.it/view/MSS_Borg.lat.285">Vat. Borg. lat. 285</idno>
                     </msIdentifier>
+
+                    <msContents>
+                      <summary>A collection of letters addressed to <persName ref="PRS2853Borgia"/>, secretary of the Congregation de Propaganda Fide.</summary>
+                    </msContents>
+
+                    <physDesc>
+                      <additions>
+                        <list>
+                          <item xml:id="a1">
+                            <locus from="352r" to="353r" facs="0359"/>
+                            <note>A letter in Italian from <persName ref="PRS9535tobiyaG"/> to <persName ref="PRS2853Borgia"/>, dated <date>19 December 1788</date>, describing the provenance and contents of <ref type="mss" corresp="BAVlat285#a1">Vat. Borg. et. 24</ref>, a scroll containing <ref type="work" corresp="LIT1758Lefafa#a1">Lǝfāfa ṣǝdq</ref>.</note>
```

Comment:
Is the manuscript reference given in ref correct? It seems to refer to exactly this addition here.

Reply:
Aha, good catch, I copied the ref from the other record but forgot to change the ID.